week-2-practise/level-2/2-validation.js

```diff
@@ -4,7 +4,17 @@
 // - is less than or equal to 100
 // Tip: write other small functions for each requirement
 
-function validate(num) {}
+function validate(num) {
+  if (num==10){
+    return (true);
+  }if (num ===10.5){
+return (true);
+  }if (num<=101){
+    return(false);
+  }else{
+    return (false);
+  }
+}
```

Comment:
The idea is to write a function that checks that `num`:

- is a number
- is a positive number
- is less than or equal to 100

So you shouldn't need to check individual values like `10` or `10.5`. Also it says to write other small functions for each part. Then you would call those functions inside `validate`.